README.md

```diff
@@ -0,0 +1,21 @@
+# Utility scripts to manage the Platform.sh Template projects.
```

Comment:
Yay, better README!  

No period at the end of titles, though.  Or colons on the others.  Also, leave one blank line between the header and the content below it.